src/core/templatetags/sidebar.py

```diff
@@ -170,6 +171,27 @@ def get_part_context(self):
         }
 
 
+class Comment(SidebarPart):
+    template_name = "tags/sidebar/parts/comment.html"
+
+    def is_visible(self):
+        request = self.context["request"]
+        page = self.context.get("self")
+        if not flag_is_active(request, "new_sidebar"):
+            return False
+        return bool(isinstance(page, NewsPage) and page.allow_comments)
```

Comment:
Just a slight suggestion on reordering the function so it reads a little easier. The principle here is only defining the variables as we need them.
```suggestion
    def is_visible(self):
        request = self.context["request"]
        if not flag_is_active(request, "new_sidebar"):
            return False
        
        page = self.context.get("self")
        return bool(isinstance(page, NewsPage) and page.allow_comments)
```